src/components/Typography.js

```diff
@@ -41,8 +41,9 @@ export const I = styled.i`
 export const A = styled.a`
   cursor: pointer;
   text-decoration: none;
-  border-bottom: 1px solid ${p => p.theme.colors.link};
-  color: ${p => p.theme.colors.link};
+  font-weight: ${p => (p.bolded ? 600 : 400)};
+  border-bottom: 1px solid ${p => (p.color ? p.theme.colors.primary : p.theme.colors.link)};
+  color: ${p => (p.color ? p.color === "primary" && p.theme.colors.primary || p.color : p.theme.colors.link)};
```

Comment:
lol I think my suggestion caused a linting error :angrycry:
can you add a bracket here?
```suggestion
  color: ${p => (p.color ? (p.color === "primary" && p.theme.colors.primary) || p.color : p.theme.colors.link)};
```